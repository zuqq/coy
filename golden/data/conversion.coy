fn main() -> () {
    print!("{}\n", 12.5 as i64);
    print!("{}\n", 1250 as f64)
}
