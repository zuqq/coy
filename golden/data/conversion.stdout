12
1250.000000
